README.md

```diff
@@ -267,12 +267,18 @@ To manually ssh to the *Deployer Concourse*, learn its IP via AWS console and
 download the `id_rsa` file from the s3 state bucket. You will need
```

Comment:
The line above this could be amended now to exclude "learn its IP via AWS console".